debezium-server-voyager/debezium-server-voyager-dist/src/main/resources/distro/run.sh

```diff
@@ -13,12 +13,26 @@ if [ -z "$1" ]; then
     echo "Argument missing. Provide application.properties file path as argument";
     exit 1;
 fi
+
 PROPERTIES_FILE_PATH="$1"
 if [ ! -f "$PROPERTIES_FILE_PATH" ]; then
     echo "$PROPERTIES_FILE_PATH does not exist."
     exit 1;
 fi
 
+if [ -z "$2" ]; then
+    echo "No YB or PG connector path provided.";
```

Comment:
Let's not design this as a positional argument but rather as an optional argument.  
A good default is to pick up the pg_connector. one can override it by setting an env variable.

That way, even the code gets simplified while invoking it in voyager. 
```
if targetDBexporterRole and !useLogicalReplicationConnector: 
    debezium_dist_dir = 1.9.5
else: 
    debezium_dist_dir = default path 
    
if targetDBexporterRole and useLogicalReplicationConnector : 
    set env var
```

Reply:
I now see the reason why it can't be an optional arg (because we use the same script for 1.9.5 but there we won't have the connector paths logic.)

Reply:
Let's call the argument `YB_OR_PG_CONNECTOR_PATH` here as well.